### PR TITLE
Reduce font size of success story card titles on mobile view

### LINKED ISSUE #1233: Edit font size in mobile view in Success story page
**Describe the bug**
The font size in mobile view on the Success story page is too large

**To Reproduce**
1. Go to 'https://sefglobal.org/success_stories/'

**Expected behavior**
The text should be larger than the sub-text, but not too large in the mobile

**Screenshots**
image

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by dave-v0**
## Purpose

The purpose of this PR is to fix #1233 

## Goals

Making the font smaller in mobile view in success stories page

## Approach

Decreasing font size in mobile view by adding media queries

### Screenshots

image

  
### Preview Link

https://pr-1260-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by dave-v0** [comment]
@anjula-sack @Piumal1999 Can you please review this?

File: success_stories/styles.css [hunk 1]
```diff
@@ -207,4 +206,10 @@ hr {
 .skeleton-storycard-text:last-child {
     margin-bottom: 0;
     width: 80%;
+}
+
+@media screen and (max-width: 720px) {
+    .story-section .highlight {
+        font-size: 20px;
+    }
 }
```
Comment (Piumal1999) [inline success_stories/styles.css]: Add EOF
```suggestion
}

```

**Review by Piumal1999 — CHANGES_REQUESTED** [review]

**Review by anjula-sack — APPROVED** [review]
Thank you! @DilinaBandara

**Review by Piumal1999 — APPROVED** [review]
Good job @DilinaBandara 
Thank you

**Pull request merged**